core/src/main/java/nu/yona/server/analysis/entities/DayActivity.java

```diff
@@ -212,4 +209,9 @@ public static DayActivity createInstance(UserAnonymized userAnonymized, Goal goa
 		return new DayActivity(UUID.randomUUID(), userAnonymized, goal, startOfDay, new ArrayList<Activity>(),
 				new ArrayList<Integer>(IntervalActivity.SPREAD_COUNT), 0, true, false);
 	}
+
+	public static DayActivity createInstanceZeroActivity(UserAnonymized userAnonymized, Goal goal, ZonedDateTime startOfDay)
```

Comment:
Isn't it better to create talk about inactivity instead of ``zeroActivity``? Inactivity is just a plain English word, understood by everyone.